WordPress/src/main/java/org/wordpress/android/ui/media/MediaSettingsActivity.java

```diff
@@ -651,6 +651,11 @@ private void setupImageSizeSeekBar() {
 
         int imageSizeKey = Arrays.asList(mImageSizeKeyArray).indexOf(mEditorImageMetaData.getSize());
 
+        //if the image size is something we can't recognize, default to the full size
```

Comment:
It seems that there is at least one value that is expected but we don't test for: the empty string as [mentioned in the comment here](https://github.com/wordpress-mobile/WordPress-Android/blob/5fa249d23643993354eabf86c19a438ae2351787/libs/editor/WordPressEditor/src/main/java/org/wordpress/android/editor/MetadataUtils.java#L37-L37).

If that's indeed an expected value, what do you think about explicitly guard for it with an `if` statement? Thanks!

Reply:
Good catch! An empty string is indeed, not an expected value, the default value will always be `size-full`. All amend the comment.